Don't kill implicit if stop failed.

praveenkumar opened the pull request:
Stop is use for graceful shutdown and
it shouldn't call implicit kill function
if shutdown doesn't happen but return an error.

Current shutdown waiting period is 5 sec which is
way to less in term of shuting down coreos VM, so
this is increased to 120 sec.

cfergeau reviewed (changes requested):
Both changes look good to me, however I would prefer  this is split in 2 patches, one for the kill removal, and the other for extending the wait.

praveenkumar committed:
Don't kill implicit if stop failed.

Stop is use for graceful shutdown and
it shouldn't call implicit kill function
if shutdown doesn't happen but return an error.

praveenkumar committed:
Increase stop wait time from 5s to 180s

Current shutdown waiting period is 5 sec which is
way to less in term of shuting down coreos VM, so
this is increased to 120 sec.

praveenkumar commented:
@cfergeau updated and merged.
